resources/views/emails/membership/membershipUnderPaid.blade.php

```diff
@@ -4,9 +4,9 @@
 @content('emails.membership.membershipUnderPaid', 'main')
 
 
-Thank you for your membership payment to {{ config('branding.space_name') }}, however your payment was below our require minimum amount of **@money($minimumAmount, 'GBP')** for membership. In order to complete you membership your payments must be equal or above the minimum amount. 
+Thank you for your membership payment to {{ config('branding.space_name') }}. Unfortunately your payment must be at least **@money($minimumAmount, 'GBP')** for membership. In order to complete your membership, please make a payment of @money($minimumAmount, 'GBP') or more, within the next few days, using your payment reference.
```

Comment:
" within the next few days,"
not sure about needing that, this email is for new members, those in the "Awaiting Payment" role, not current members

Reply:
good spot, thanks, sorted